Update DownloadYourData exports for `decidim-initiatives`

andreslucena opened the pull request:
#### :tophat: What? Why?

This PR updates the DownloadYourData exports for decidim-initiatives models.

On this case what I'm doing is: 

- Doing the same strategy as in #13895, I inherit the Serializer with the DownloadYourDataSerializer to make it more maintainable 

#### :pushpin: Related Issues
 
- Related to #13527

#### Testing

- Everything should be green
- There shouldn't be new fields to add here 

:hearts: Thank you!

alecslupu reviewed (changes requested):
I have tried it, and there is no export generated. Apparenty, you've forgot to add 

```
  participatory_space.data_portable_entities = [
    "Decidim::Initiative"
  ]
```
to `decidim-initiatives/lib/decidim/initiatives/participatory_space.rb`

andreslucena commented:
> I have tried it, and there is no export generated. Apparenty, you've forgot to add

Good one! I fixed it on [bfde8f6](https://github.com/decidim/decidim/pull/13961/commits/bfde8f6c660e52dcf48062f6f9a11c9948e6fe61)

I added a spec just in case, and also for checking out the missing i18n keys here

Finally, I also fixed a bug like the one you mentioned in https://github.com/decidim/decidim/pull/13958#pullrequestreview-2579368131 

This is ready for another round 

![bring-it-on](https://github.com/user-attachments/assets/1e095c00-ac46-4c10-8f27-377fce73c647)

alecslupu commented on decidim-initiatives/spec/serializers/download_your_data_initiative_serializer_spec.rb:
```diff
@@ -0,0 +1,134 @@
+        expect(serialized[:type]).to include(title: initiative.type.title)
+      end
+
+      it "includes the authors' ids" do
+        expect(serialized[:authors]).to include(id: initiative.author_users.map(&:id))
+      end
+
+      it "includes the authors' names" do
+        expect(serialized[:authors]).to include(name: initiative.author_users.map(&:name))
+      end
+
+      it "includes the area id" do
```
The pipeline fails having those 2 specs here...  I would suggest changing the spec to ... 
```suggestion
      it "excludes the authors" do
        expect(serialized).not_to have_key(:authors)
      end

      it "includes the area id" do
```

alecslupu reviewed (changes requested):
The specs are failing  Could you have a look on my suggestion? ...

alecslupu reviewed (approved):
:+1: 

Merging with codecov failing pipeline.
